src/backend/logger/__tests__/logfile.test.ts

```diff
@@ -25,29 +25,33 @@ describeSkipOnWindows('logger/logfile.ts', () => {
     tmpDir.removeCallback()
   })
 
-  test('createNewLogFileAndClearOldOnes fails because logDir does not exist', () => {
-    const spyAppGetPath = jest.spyOn(app, 'getPath').mockReturnValue('invalid')
-    const spyOpenSync = jest.spyOn(graceful_fs, 'openSync')
-
-    logfile.createNewLogFileAndClearOldOnes()
-
-    const year = `${new Date().getFullYear()}`
-
-    expect(spyOpenSync).toBeCalledWith(
-      expect.stringContaining(`invalid/${year}-`),
-      'w'
-    )
-    expect(spyAppGetPath).toBeCalledWith('logs')
-    expect(logError).toBeCalledWith(
-      [
-        expect.stringContaining(`Open invalid/${year}-`),
-        expect.objectContaining(
-          Error(`ENOENT: no such file or directory, open 'invalid/${year}-`)
-        )
-      ],
-      { prefix: 'Backend', skipLogToFile: true }
-    )
-  })
+  // FIXME: this test for some reason is correct in CI but `app.getPath` returning 'invalid'
```

Comment:
Maybe run it if the ENV for CI is true.
Something like:
`if (process.env.GITHUB_ACTIONS) {`

Reply:
one thing I'm seeing now is that this is testing a case that will never happen

because from the getPath method, when it returns a string it's a valid path and when it's invalid it throws an error (that we are not catching), it should never return an invalid path https://www.electronjs.org/docs/latest/api/app#appgetpathname

we seem to be handling an invalid path incorrectly

I'd rather remove this test to avoid a false sense of correctness, we should actually do a try/catch in `createNewLogFileAndClearOldOnes` and return if the path is invalid